asapdiscovery-data/asapdiscovery/data/dask_utils.py

```diff
@@ -55,7 +56,7 @@ def set_dask_config():
 def actualise_dask_delayed_iterable(
     delayed_iterable: Iterable,
     dask_client: Optional[Client] = None,
-    errors: str = "raise",
+    errors: str = DaskFailureMode.RAISE.value,
```

Comment:
Needs adding to the doc string, and should the type be `DaskFailureMode` and then you can pass the value to dask?